lib/fastlane/plugin/wpmreleasetoolkit/helper/android/android_version_helper.rb

```diff
@@ -328,24 +304,26 @@ def self.bump_version_release(app)
 
         # Update the `version.properties` file with new `versionName` and `versionCode` values
         #
-        # @param [String] app The name of the app to be used for beta and alpha version update
+        # @param [String] app (UNUSED/DEPRECATED) The name of the app to be used for beta and alpha version update
+        #                     Unused anymore, will be removed in future breaking update
         # @param [Hash] new_version_beta The version hash for the beta, containing values for keys "name" and "code"
         # @param [Hash] new_version_alpha The version hash for the alpha , containing values for keys "name" and "code"
         #
         def self.update_versions(app, new_version_beta, new_version_alpha)
-          if properties_file_exists
-            new_version_name_beta_key = "#{app}.versionName"
-            new_version_code_beta_key = "#{app}.versionCode"
-            Action.sh('./gradlew', 'updateVersionProperties', "-Pkey=#{new_version_name_beta_key}", "-Pvalue=#{new_version_beta[VERSION_NAME]}")
-            Action.sh('./gradlew', 'updateVersionProperties', "-Pkey=#{new_version_code_beta_key}", "-Pvalue=#{new_version_beta[VERSION_CODE]}")
-
-            unless new_version_alpha.nil?
-              new_version_name_alpha_key = "#{app}.#{alpha_flavor_name}.versionName"
-              new_version_code_alpha_key = "#{app}.#{alpha_flavor_name}.versionCode"
-
-              Action.sh('./gradlew', 'updateVersionProperties', "-Pkey=#{new_version_name_alpha_key}", "-Pvalue=#{new_version_alpha[VERSION_NAME]}") unless new_version_alpha.nil?
-              Action.sh('./gradlew', 'updateVersionProperties', "-Pkey=#{new_version_code_alpha_key}", "-Pvalue=#{new_version_alpha[VERSION_CODE]}") unless new_version_alpha.nil?
+          if File.exist?(version_properties_file)
```

Comment:
Nit, but would this be a bit cleaner?

```suggestion
          return unless File.exist?(version_properties_file)
```

Reply:
Except there's an else clause to that `if` 😛